src/app/tests/suites/TestLevelControlWithOnOffDependency.yaml

```diff
@@ -0,0 +1,225 @@
+              - name: "level"
+                value: 1
+              - name: "transitionTime"
+                value: 0
+              - name: "optionMask"
+                value: 0
+              - name: "optionOverride"
+                value: 0
+
+    - label: "Wait 100 ms"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 100
+
+    - label: "Reads CurrentLevel attribute from DUT"
+      command: "readAttribute"
+      attribute: "current level"
+      response:
+          value: 1
+
+    - label: "Write OnOffTransitionTime attribute"
+      command: "writeAttribute"
+      attribute: "on off transition time"
+      arguments:
+          value: 0
+
+    - label: "Read OnOffTransitionTime attribute"
+      command: "readAttribute"
+      attribute: "on off transition time"
+      response:
+          value: 0
+
+    - label: "Write OnLevel attribute"
+      command: "writeAttribute"
+      attribute: "on level"
+      arguments:
+          value: 254
+
+    - label: "Read OnLevel attribute"
+      command: "readAttribute"
+      attribute: "on level"
+      response:
+          value: 254
+
+    - label: "Read MinValue attribute"
+      command: "readAttribute"
+      attribute: "on off transition time"
+      response:
+          value: 1
+
+    - label: "Send On Command"
+      cluster: "On/Off"
+      command: "On"
+
+    - label: "Check on/off attribute value is true after on command"
+      cluster: "On/Off"
+      command: "readAttribute"
+      attribute: "OnOff"
+      response:
+          value: 1
+          
+    - label: "Wait OnOffTransitionTime"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 100
+
+    - label: "If OnLevel is defined, check CurrentLevel is OnLevel value"
+      command: "readAttribute"
+      attribute: "current level"
+      response:
+          value: 254
+
+    - label: "Send Off Command"
+      cluster: "On/Off"
+      command: "Off"
+
+    - label: "Check on/off attribute value is false after off command"
+      cluster: "On/Off"
+      command: "readAttribute"
+      attribute: "OnOff"
+      response:
+          value: 0
+          
+    - label: "Wait OnOffTransitionTime"
+      cluster: "DelayCommands"
+      command: "WaitForMs"
+      arguments:
+          values:
+              - name: "ms"
+                value: 0
+
+    - label: "If OnLevel is defined, check CurrentLevel is min value"
+      command: "readAttribute"
+      attribute: "current level"
+      response:
+          value: 1
+
+    - label: "Sends a MoveToLevel command to set current level to a mid value"
+      command: "MoveToLevel"
+      arguments:
+          values:
+              - name: "level"
+                value: 127
+              - name: "transitionTime"
+                value: 0
+              - name: "optionMask"
```

Comment:
Same as before set optionMask and optionOverride 1

Reply:
Done.